package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "flowcommonweb",
-  "version": "1.0.0",
+  "version": "0.0.1",
   "description": "**When you export a Flow project to HTML, React, or anything Web-related, Flow includes a set of helper classes and extensions that help manage and control your animations. This page provides and overview and a link to each of these files.**",
-  "main": "Animation.js",
+  "main": "ToggleButton.js",
```

Comment:
What is the main file in `package.json` like what role/purpose does it have? Is it the file that shows up some where on npm as a sample file or something? 

I feel like the main one should be player?